django/src/rdwatch/utils/satellite_bands.py

```diff
@@ -31,113 +31,96 @@ def get_bands(
     if constellation.slug not in ('L8', 'S2'):
         raise ValueError(f"Unsupported constellation '{constellation.slug}'")
 
-    page: None | int = 1
-
-    while page is not None:
-        results = stac_search(
-            constellation.slug,
-            timestamp,
-            bbox,
-            timebuffer=timebuffer or timedelta(hours=1),
-            page=page,
-        )
-
-        if 'features' not in results:
-            logger.warning("Malformed STAC response: no 'features'")
-            return
-
-        if 'links' not in results:
-            logger.warning("Malformed STAC response: no 'links'")
-            return
-
-        next_page = None
-        for link in results['links']:
-            match link:
-                case {'rel': 'next'}:
-                    next_page = page + 1
-                    break
-        page = next_page
-
-        for feature in results['features']:
-            if 'assets' not in feature:
-                logger.warning("Malformed STAC response: no 'assets'")
+    results = stac_search(
+        constellation.slug,
+        timestamp,
+        bbox,
+        timebuffer=timebuffer or timedelta(hours=1),
+    )
+
+    if 'features' not in results:
+        logger.warning("Malformed STAC response: no 'features'")
+        return
+
```

Comment:
A note about this file - it looks like a lot has changed here, but all I did was remove the while loop and the conditional that checks the `links` field for pagination info. The large diff is due to the de-indent I had to do after removing the while loop.